src/main/java/com/blackducksoftware/integration/jira/common/BlackDuckDataHelper.java

```diff
@@ -116,6 +123,35 @@ public ProjectVersionWrapper getProjectVersionWrapper(final VersionBomComponentV
         return new ProjectVersionWrapper();
     }
 
+    public Optional<NotificationType> getNotificationTypeFromPolicyRule(final PolicyRuleView policyRule) {
+        final Optional<PolicySummaryStatusType> optionalPolicySummaryStatusType = getPolicySummaryStatusTypeFromRule(policyRule);
+        if (optionalPolicySummaryStatusType.isPresent()) {
+            switch (optionalPolicySummaryStatusType.get()) {
+                case IN_VIOLATION:
+                    return Optional.of(NotificationType.RULE_VIOLATION);
+                case NOT_IN_VIOLATION:
+                    return Optional.of(NotificationType.RULE_VIOLATION_CLEARED);
+                case IN_VIOLATION_OVERRIDDEN:
+                    return Optional.of(NotificationType.POLICY_OVERRIDE);
+            }
+        }
+        return Optional.empty();
+    }
+
+    private Optional<PolicySummaryStatusType> getPolicySummaryStatusTypeFromRule(final PolicyRuleView policyRule) {
+        // TODO remove when blackduck-common-api supports this field
+        if (policyRule.getEnabled()) {
+            final String jsonFieldName = "policyApprovalStatus";
+            final JsonObject policyObject = policyRule.getJsonElement().getAsJsonObject();
```

Comment:
We could also use JsonParse since we can construct a JsonPath for this field. But if we keep the current approach. We may want to check if policyRule.getJsonElement() is null and if it is not null if that value is actually a JsonObject